RFS/src/main/java/com/rfs/common/LuceneDocumentsReader.java

```diff
@@ -135,18 +153,44 @@ protected DirectoryReader wrapReader(DirectoryReader reader, boolean softDeletes
         return reader;
     }
 
-    protected Document getDocument(IndexReader reader, int docId, boolean isLive) {
+    protected RfsLuceneDocument getDocument(IndexReader reader, int docId, boolean isLive) {
         try {
             Document document = reader.document(docId);
-            BytesRef sourceBytes = document.getBinaryValue("_source");
-            String id;
+            String id = null;
+            BytesRef sourceBytes = null;
             try {
-                var idValue = document.getBinaryValue("_id");
-                if (idValue == null) {
+                for (var field : document.getFields()) {
+                    log.atDebug().setMessage("Field: {}").addArgument(field.name()).log();
```

Comment:
This seems like trace level logging, is it still needed?  If you want to keep it, lets bump to trace level and could you include the index name / doc id with the field.

Reply:
Oops, that was for debugging.  Will remove.